application/templates/static_site/static_pages/ethnic_groups.html

```diff
@@ -85,21 +85,21 @@ <h2 class="govuk-heading-s govuk-!-margin-bottom-1">Other ethnic group</h2>
 
         <p class="govuk-body">It is recognised that these ethnic groups do not represent how all people identify. People are encouraged to write in their ethnicity using their own words if they don’t identify with any groups in the list. </p>
 
-        <p class="govuk-body">There has been an ethnicity question in the Census since 1991. It has been updated at every new Census since then. There are separate questions in the Census about <a class="govuk-link" href="https://www.ons.gov.uk/peoplepopulationandcommunity/culturalidentity/ethnicity/articles/ethnicityandnationalidentityinenglandandwales/2012-12-11#national-identity-in-england-and-wales">national identity</a> and <a class="govuk-link" href="https://www.ons.gov.uk/peoplepopulationandcommunity/culturalidentity/religion">religion</a>.</p>
+        <p class="govuk-body">There has been an ethnicity question in the Census since 1991. It has been updated at every new Census since then. There are separate questions in the Census about <a class="govuk-link" href="https://www.ons.gov.uk/peoplepopulationandcommunity/culturalidentity/ethnicity/articles/ethnicityandnationalidentityinenglandandwales/2012-12-11#national-identity-in-england-and-wales" data-on="click" data-event-category="External link clicked" data-event-action="Ethnic groups link" data-event-label="national identity">national identity</a> and <a class="govuk-link" href="https://www.ons.gov.uk/peoplepopulationandcommunity/culturalidentity/religion" data-on="click" data-event-category="External link clicked" data-event-action="Ethnic groups link" data-event-label="religion">religion</a>.</p>
```

Comment:
Possibly as a convention, we should use something like the title of the page we're linking to as the event label, rather than the link text (`national identity`)?  That might make it easier to tell what the link is when reviewing the analytics, plus be a bit more stable (in case the link text changes)?

Reply:
Done 👍